### PR TITLE
fix(curriculum): use numbers in english challenges

### PR DESCRIPTION
**Pull request by moT01**
We were getting gatsby warnings about types in the english curriculum:

Screenshot 2024-01-04 at 9 51 23 AM

This fixes the challenge files and forces the types.

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by moT01** [comment]
Want to take a quick look at this one @Sembauke - I changed some of the challenge schema. It's saying to notify mobile team.

**Comment by moT01** [comment]
blocked until we get the go-ahead from the mobile team

**Review by Nirajn2311 — APPROVED** [review]
LGTM and doesn't break the mobile app

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**
